libnavui-dropin/api/current.txt

```diff
@@ -110,6 +110,7 @@ package com.mapbox.navigation.dropin {
 
   @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public final class ViewOptionsCustomization {
     ctor public ViewOptionsCustomization();
+    method public com.mapbox.navigation.base.formatter.DistanceFormatterOptions? getDistanceFormatterOptions();
```

Comment:
Is this supposed to be here?

Reply:
@abhishek1508 forgot to add these in #6222, so why not add it now.